app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -5,11 +5,14 @@
   shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
   items ||= []
 
-  classes = %w[gem-c-document-list]
-  classes << "gem-c-document-list--no-underline" if local_assigns[:remove_underline]
-  classes << "gem-c-document-list--no-top-border" if local_assigns[:remove_top_border]
-  classes << "gem-c-document-list--no-top-border-first-child" if local_assigns[:remove_top_border_from_first_child]
-  classes << shared_helper.get_margin_bottom
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+  component_helper.add_class("gem-c-document-list")
+  component_helper.add_class("gem-c-document-list--no-underline") if local_assigns[:remove_underline]
+  component_helper.add_class("gem-c-document-list--no-top-border") if local_assigns[:remove_top_border]
+  component_helper.add_class("gem-c-document-list--no-top-border-first-child") if local_assigns[:remove_top_border_from_first_child]
+  margin_bottom = shared_helper.get_margin_bottom
+  component_helper.add_class(margin_bottom) if margin_bottom
```

Comment:
I think this `if margin_bottom` check might be redundant because `shared_helper.get_margin_bottom` always returns a value, so I think these two lines could be written more simply as

```
component_helper.add_class(shared_helper.get_margin_bottom)
```

Reply:
@andysellick Thanks, will update and then merge 👍